Soccar [Client]/src/nl/soccar/ui/fx/models/MapUiFx.java

```diff
@@ -39,40 +41,56 @@
      */
     public MapUiFx(GameCanvasFx canvas, Map model) {
         super(canvas, model);
-
-        addWalls();
     }
 
-    private void addWalls() {
+    public void addWalls() {

```

Comment:
Op dit moment worden de muren in GameViewFXMLController toegevoegd NA het aanmaken van het MapUiFx object zodat deze worden getekend boven de map.